src/sentenceCount.ts

```diff
@@ -0,0 +1,8 @@
+const sentence = 'Esta es una frase de prueba, que tiene 10 palabras';
+
+function wordsCount(): number {
+  const sentenceArray = sentence.split(' ');
+  return sentenceArray.length;
+}
+
+console.log(`The sentence '${sentence}' has ${wordsCount()} words`);
```

Comment:
Podríamos hacer que la función reciba la frase por parámetro, por si quisiéramos testear con varias frases distintas.

```suggestion
function wordsCount(sentence: string): number {
  const sentenceArray = sentence.split(' ');
  return sentenceArray.length;
}

console.log(`The sentence '${sentence}' has ${wordsCount(sentence)} words`);
```